packages/flutter/lib/src/gestures/long_press.dart

```diff
@@ -142,8 +142,8 @@ class LongPressEndDetails {
 /// [postAcceptSlopTolerance] constructor argument is specified.
 ///
 /// [LongPressGestureRecognizer] competes on pointer events of [kPrimaryButton]
-/// only when it has at least one non-null callback. If it has no callbacks, it
-/// is a no-op.
+/// or [kSecondaryButton] only when it has at least one non-null callback. If it
```

Comment:
[LongPressGestureRecognizer] may compete on pointer events of [kPrimaryButton] and/or
[kSecondaryButton] if at least one corresponding callback is non-null.

Reply:
Done.